### ISSUE TITLE
Stores my password in plain text

### ISSUE THREAD

**Issue by dylan-cooper** [deyvidm/keychain#1]
Hi, I was using your app and I input my password.  I found out later that it stored it in plain text! You may want to fix this by using some encryption.

**Comment by deyvidm** [comment]
Yup thanks for pointing that out, I've thought about it as well. Any ideas how I should resolve this? If I encrypt on the user's machine, what would I do with the key? Wouldn't it also need to be in plaintext?
